gematria/model/python/token_model_flags.py

```diff
@@ -114,25 +157,40 @@ def get_tokens_from_command_line_flags(  #
     or None when no such file is specified. The returned list is sorted and
     contains each token at most once.
   """
-  if not _TOKEN_FILE.value:
+  tokens_from_file = get_lines_from_file(_TOKEN_FILE.value)
+  if tokens_from_file is None:
     return None
 
-  tokens = set(model_tokens)
-  with tf.io.gfile.GFile(_TOKEN_FILE.value, 'r') as f:
-    for line in f:
-      line = line.strip()
-      if not line or line.startswith('#'):
-        continue
-      tokens.add(line)
+  return sorted(set(model_tokens) | set(tokens_from_file))
+
+
+def get_annotation_names_from_command_line_flags() -> Sequence[str]:
+  """Returns the list of annotation names used in the model.
+
+  When the command-line flag --gematria_annotation_names_file is used, returns a
+  sorted list of annotation_names from this file. The input file is expected to
```

Comment:
Very nit: s/annotation\_names/annotation names/

Reply:
Ah, I'm wondering how that happened :)